src/context/cdhashmap.h

```diff
@@ -211,8 +211,7 @@ class CDOhash_map : public ContextObj
     {
       // "Initializing" map insertion: this entry will never be
       // removed from the map, it's inserted at level 0 as an
-      // "initializing" element.  See
-      // CDHashMap<>::insertAtContextLevelZero().
+      // "initializing" element.
```

Comment:
From a cursory look, it seems that we might be able to remove the `atLevelZero` argument. Could you give that a try?